docs/advanced/symbol-server-compatibility.md

```diff
@@ -139,7 +173,17 @@ Casing rules for Symbol Server are mixed:
 The presence of a `index2.txt` in the root indicates two tier structure where
 the first two characters are prepended to the path as an additional folder. So
 `foo.exe/542D5742000f2000/foo.exe` is stored as
-`fo/foo.exe/542D5742000f2000/foo.exe`.
+`fo/foo.exe/542D5742000f2000/foo.exe`. Note that symbolicator does not probe
+for the `index2.txt` file. You need to be explicit in configuring it.
+
+Symbol bundles are only supported for PE/PDB files with the following format:
```

Comment:
```suggestion
Source bundles are only supported for PE/PDB files with the following format:
```

Might wanna clarify here that the source bundle is always keyed after the PDB and never the EXE.